mobile/src/androidTest/java/org/openhab/habdroid/ui/BasicWidgetTest.java

```diff
@@ -45,48 +49,25 @@
 public class BasicWidgetTest {
 
     @Rule
-    public ActivityTestRule<OpenHABMainActivity> mActivityTestRule = new ActivityTestRule<>(OpenHABMainActivity.class);
+    public ActivityTestRule<OpenHABMainActivity> mActivityTestRule = new ActivityTestRule<>
+            (OpenHABMainActivity.class, true, false);
 
     private IdlingResource mProgressbarIdlingResource;
 
-    @Test
-    public void openHABMainActivityTest2() throws InterruptedException {
-        // click next
-        ViewInteraction appCompatImageButton = onView(
-                allOf(withId(R.id.next),
-                        childAtPosition(
-                                allOf(withId(R.id.bottomContainer),
-                                        childAtPosition(
-                                                withId(R.id.bottom),
-                                                1)),
-                                3),
-                        isDisplayed()));
-        appCompatImageButton.perform(click());
-
-        // click next
-        ViewInteraction appCompatImageButton2 = onView(
-                allOf(withId(R.id.next),
-                        childAtPosition(
-                                allOf(withId(R.id.bottomContainer),
-                                        childAtPosition(
-                                                withId(R.id.bottom),
-                                                1)),
-                                3),
-                        isDisplayed()));
-        appCompatImageButton2.perform(click());
+    @Before
+    public void setup() {
+        PreferenceManager
+                .getDefaultSharedPreferences(InstrumentationRegistry.getTargetContext())
+                .edit()
+                .putString(Constants.PREFERENCE_SITEMAP, "")
+                .putBoolean(Constants.PREFERENCE_FIRST_START, false)
```

Comment:
You should also enable demo mode. Its disabled by default in #562

Reply:
I would do it in #562 if this PR is merge before it, or otherwise add it once #562 is merged, ok? Doing that keeps both PRs clean I think :]